robotiq_cpp_control/CMakeLists.txt

```diff
@@ -0,0 +1,92 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(robotiq_cpp_control)
+
+set(GCC_COVERAGE_COMPILE_FLAGS "-g -Wno-enum-compare -std=c++11 -Wenum-compare -Wformat-security")
+set(GCC_COVERAGE_LINK_FLAGS    "-Wformat-security")
+
+set( CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${GCC_COVERAGE_COMPILE_FLAGS}" )
+set( CMAKE_EXE_LINKER_FLAGS  "${CMAKE_EXE_LINKER_FLAGS} ${GCC_COVERAGE_LINK_FLAGS}" )
+
+find_package(catkin REQUIRED COMPONENTS
+  robotiq_s_model_control
+  robotiq_c_model_control
+  roscpp
+  std_msgs
+  message_generation
+)
+
+## Generate messages in the 'msg' folder
+add_message_files(
+  FILES
+  RSGripperCommand.msg
+  RCGripperCommand.msg
+)
+
+## Generate services in the 'srv' folder
```

Comment:
Can you clear out the auto-generated (commented out) lines in the `CMakeLists.txt`